src/twisted/conch/ssh/keys.py

```diff
@@ -589,7 +617,10 @@ def _guessStringType(cls, data):
             return 'public_lsh'
         elif data.startswith(b'('):
             return 'private_lsh'
-        elif data.startswith(b'\x00\x00\x00\x07ssh-') or data.startswith(b'\x00\x00\x00\x13ecdsa-'):
+        elif data.startswith(b'\x00\x00\x00\x0bssh-'):  # ED25519
```

Comment:
The data will continue `ssh-ed25519` right? Might be easier to read if you include that in the string.